lib/generators/doorkeeper/templates/initializer.rb

```diff
@@ -343,15 +333,38 @@
   # @param authorized_token [Doorkeeper::AccessToken]
   #   Bearer token used to authorize the request
   #
-  # Keep in mind, that in case the block returns `nil` or `false` introspection response
-  # doesn't have 401 status code and some descriptive body, you'll get 200 with
-  # { "active": false } body as stated in the RFC 7662 section 2.2. Introspection Response.
+  # In case the block returns `nil` or `false` introspection responses with 401 status code
+  # when using authorized token to introspect, or you'll get 200 with { "active": false } body
+  # when using authorized client to introspect as stated in the
+  # RFC 7662 section 2.2. Introspection Response.
+  #
+  # Using with caution:
+  # Keep in mind that these three parameters pass to block can be nil as following case:
```

Comment:
Adding comments to explain when `allow_token_introspection` block's parameters get `nil`.